### PR TITLE
Hearthkin Part Two: Now Fully Functional

### PR DESCRIPTION
**Pull request by MortoSasye**
## About The Pull Request

Adds the Hearthkin's hearth and related objects, etc, finally.

Updated some sprites while I was at it too.

More pr's will follow unavoidably to add bits and details here and there.

Full Map:

image

image

Key areas:
<details>

image

image

image

image

image

image

image

</details>

## Why It's Good For The Game

WE HAVE OUTLANDERS TO K*LL, KIN

## Changelog

:cl: MortoSasye And Kaostico
add: Added a fully functional Hearthkin's Hearth.
add: Added Hearthkin related ruins: Abandoned Hearth, Ancient Temple, Magical Hot Springs.
add: Added the catnip plant and honeysuckle plant.
add: Added saunas to the game.
add: Added primitive tools, including surgery ones.
add: Added towels and towel bins to the game.
add: Added exclusive hearthkin recipes: fur rugs, bronze calling horn, and the warning rune stone.
add: Added the automapper system, and its unit test.
fix: Linter now checks on modular_doppler if dm files aren't included on tgstation.dme
fix: Linter now works on unit test's subdirectories correctly
image: Updated the sprites for the tailored dress, the tunic, and the ferroniere.
/:cl:

### CODE REVIEW

File: modular_doppler/modular_mapping/ruins/icemoon.dm [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*----- Template for ruins, prevents needing to re-type the filepath prefix -----*/
+/datum/map_template/ruin/icemoon/underground/doppler/
+	prefix = "_maps/RandomRuins/IceRuins/doppler/"
+
+/datum/map_template/ruin/icemoon/nova/
```
Comment (honkpocket) [inline modular_doppler/modular_mapping/ruins/icemoon.dm:5]: Oversight?

Comment (MortoSasye) [reply modular_doppler/modular_mapping/ruins/icemoon.dm]: Oopsies! Fixed!

**Comment by MortoSasye** [comment]
Hold off on merging this, adding some more stuff while I see if https://github.com/DopplerShift13/DopplerShift/pull/21 gets accepted during that period.

**Comment by MortoSasye** [comment]
DO NOT MERGE. There's a game breaking bug re: crafting that is becoming a fucking nuisance to find, and it's getting late on our side.

Added:
- A calling horn, which will become functional later. *
- A rune stone warning people to not invade the hearth. * 
- Fur rugs for decoration, that work as structures. *
- Capes are now found in the wardrobe.

* Crafting recipes included, exclusively for hearthkin.

<details>

dreamseeker_F6KZeSxMh7
dreamseeker_Lrw2HPHfGV
dreamseeker_Qlqfj9U1pO
dreamseeker_Hl4hyRNBKl
dreamseeker_1vPgdZtqxe

</details>

**Comment by MortoSasye** [comment]
Ok now it can be merged, we have finished fist fighting this code for now.

Added functions to deconstruct the new items, as well as anchor/un-anchor the horn and the stones with ctrl + click

<details>

dreamseeker_XvbbvsZoXP
nVpyD8m9Sv
dreamseeker_eSysHnFD7u
dreamseeker_o7nyX7XAI8
dreamseeker_kAedz4Njjz
dreamseeker_3BPFcFUEkT
dreamseeker_8GDx1rkaDu
VQnrJFrm75

</details>

**Review by honkpocket — APPROVED** [review]

**Comment by honkpocket** [comment]
I blow my bugle.

**Pull request merged**